### PR TITLE
fix(accounts): add reference links to docs on custom roles and SSO

### PR DESCRIPTION
**Pull request by ghost**
This PR is in response to customer feedback requesting info on how to create custom roles and configure different access levels for users with SSO authentication.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx [hunk 1]
```diff
@@ -41,10 +41,14 @@ At New Relic, user <DoNotTranslate>**groups**</DoNotTranslate> contain <DoNotTra
...
 
 When you initially sign up for New Relic, your organization has some built-in role and account assignments for the default [<DoNotTranslate>**User**</DoNotTranslate> or <DoNotTranslate>**Admin**</DoNotTranslate> groups](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#groups). For example, the <DoNotTranslate>**Admin**</DoNotTranslate> group has several role assignments that give users in that group broad New Relic access, including to the higher level administrative permissions.
 
+<Callout variant="tip">
+Using SSO? See [Authentication domains: How your users log in and are managed](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/).
```
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:49]:
```suggestion
Want to control how users access New Relic? Manage authentication domains [here](https://github.com/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/).
```

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx [hunk 2]
```diff
@@ -41,10 +41,14 @@ At New Relic, user <DoNotTranslate>**groups**</DoNotTranslate> contain <DoNotTra
 * A specific [role](#roles) (a role being a set of specific, granular [permissions](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions)).
 * One or more accounts.
 
-Organizations with [Pro or Enterprise edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) can have multiple accounts in their organization, and can create custom roles and groups. Our Free and Standard editions allow only a single account in an organization, and don't let you create custom roles or groups.
+Organizations with [Pro or Enterprise edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) can have multiple accounts in their organization, and can [create custom roles and groups](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial/#roles). Our Free and Standard editions allow only a single account in an organization, and don't let you create custom roles or groups.
```
Comment (jeff-colucci) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:44]: pro or enterprise edition should be plural, right? I think the last comma in the last sentence doesn't actually need to be there, but I think that's mostly a preference call.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx [hunk 3]
```diff
@@ -77,7 +81,7 @@ A user must be in at least one group with access to a role and at least one acco
 
 Note that groups are **not** what restrict a user's New Relic permissions: it's the <DoNotTranslate>**role**</DoNotTranslate> assigned to that group that grant access to the permissions.
 
-We have two simple user groups available by default (see below). And Pro and Enterprise organizations can [create custom groups](/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model/#group-access).
+We have two simple user groups available by default (see below). And Pro and Enterprise organizations can [create custom groups](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial/#group-access).
```
Comment (jeff-colucci) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:84]: I think you can cut the And at the beginning of the second sentence.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx [hunk 4]
```diff
@@ -92,7 +96,7 @@ To edit the group a user is in, you can go to either the <DoNotTranslate>**Acces
 
 ## Roles [#roles]
 
-We offer several [default roles](#standard-roles), but organizations with Pro or Enterprise edition can create their own custom roles.
+We offer several [default roles](#standard-roles), but organizations with Pro or Enterprise edition can create their own [custom roles](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial/#roles).
```
Comment (jeff-colucci) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:99]: again, pro or enterprise edition should be plural I think

**Review by jeff-colucci — DISMISSED** [review]
Looks great overall. I saw a couple small things but once those are fixed you're good to publish without needing re-approval.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx (hunk 2 above)
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:44]:
```suggestion
Organizations with [Pro or Enterprise editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) can have multiple accounts in their organization, and can [create custom roles and groups](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial/#roles). Our Free and Standard editions allow only a single account in an organization, and don't let you create custom roles or groups.
```

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx (hunk 3 above)
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:84]:
```suggestion
We have two simple user groups available by default (see below). Pro and Enterprise organizations can [create custom groups](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial/#group-access).
```

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx (hunk 4 above)
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:99]:
```suggestion
We offer several [default roles](#standard-roles), but organizations with Pro or Enterprise editions can create their own [custom roles](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial/#roles).
```

**Comment by ally-sassman** [comment]
@jeff-colucci Thanks for your feedback!

**Pull request merged**